views/js/controller/Reporting/index.js

```diff
@@ -0,0 +1,229 @@
+            };
+            
+            var today = moment().format('YYYY-MM-DD');
+            
+            $list
+                .on('query.datatable', function() {
+                    loadingBar.start();
+                })
+                .on('load.datatable', function() {
+                    loadingBar.stop();
+                })
+                .datatable({
+                    url: serviceUrl,
+                    status: {
+                        empty: __('No reports to display!'),
+                        available: __('Available reports'),
+                        loading: __('Loading')
+                    },
+                    tools: [{
+                        id: 'download',
+                        icon: 'download',
+                        title: __('Download the selected reports to a CSV file'),
+                        label: __('Download CSV'),
+                        action: function() {
+                            notYet();
+                        }
+                    }],
+                    selectable: true,
+                    model: [{
+                        id: 'delivery',
+                        label: __('Delivery')
+                    }, {
+                        id: 'testtaker',
+                        label: __('Test Taker')
+                    }, {
+                        id: 'proctor',
+                        label: __('Proctor')
+                    }, {
+                        id: 'status',
+                        label: __('Status')
+                    }, {
+                        id: 'start',
+                        label: __('Start'),
+                        transform: function(value) {
+                            var d = new moment(value);
+                            return d.toString();
+                        }
+                    }, {
+                        id: 'end',
+                        label: __('End'),
+                        transform: function(value) {
+                            var d = new moment(value);
+                            return d.toString();
+                        }
+                    }, {
+                        id: 'pause',
+                        label: __('Pause #')
+                    }, {
+                        id: 'resume',
+                        label: __('Resume #')
+                    }, {
+                        id: 'irregularities',
+                        label: __('Irregularities')
+                    }],
+                    params:{
+                        periodStart : today,
+                        periodEnd : today
+                    }
+                }, dataset);
+            
+            //init date range picker
+            dateRangePicker(today, $container, $list);
+        }
+    };
+    
+    /**
+     * Create a data range picker for reporting index
+     * 
+     * @param {JQuery} $container
+     * @param {JQuery} $list
+     */
+    function dateRangePicker(date, $container, $list){
```

Comment:
If you're going to use this more than once, then rewrap the datatable with the datepicker into a component